index.js

```diff
@@ -135,9 +135,12 @@ module.exports = function (log, isReady, mapper) {
               Looper,
               sv.createSink(function (err) {
                 if(!flume.closed) {
-                  if(err)
-                    console.error(explain(err, 'view stream error, from:'+name))
-                  sv.since.once(build)
+                  if (err) {
+                    console.error(explain(err, `rebuilding ${name} after view stream error`))
+                    sv.destroy(() => build(-1))
```

Comment:
I've not seen this error so not sure this is the perfect place for this. BUT, it seems reasonable?

My main comment would be that there are two places now with this magic function 

```js
                    sv.destroy(() => build(-1))
```

I would like to see that defined as a function, like 
```js
function resetView () {
  sv.destroy(() => build(-1))
}
```

because then we have a clear place where we can name, comment + maintain this seeminlgy vital piece of code

Reply:
Will do, thanks.

Reply:
Done!